spec.html

```diff
@@ -42955,6 +42966,383 @@ <h1>Properties of _TypedArray_ Instances</h1>
+          1. NOTE: The order of validation and decoding in the algorithm below is not observable. Implementations are encouraged to perform them in whatever order is most efficient, possibly interleaving validation with decoding, as long as the behaviour is observably equivalent.
+          1. If _maxLength_ is 0, then
+            1. Return the Record { [[Read]]: 0, [[Bytes]]: « », [[Error]]: ~none~ }.
+          1. Let _read_ be 0.
+          1. Let _bytes_ be « ».
+          1. Let _chunk_ be the empty String.
+          1. Let _chunkLength_ be 0.
+          1. Let _index_ be 0.
+          1. Let _length_ be the length of _string_.
+          1. Repeat,
+            1. Set _index_ to SkipAsciiWhitespace(_string_, _index_).
+            1. If _index_ = _length_, then
+              1. If _chunkLength_ > 0, then
+                1. If _lastChunkHandling_ is *"stop-before-partial"*, then
+                  1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: ~none~ }.
+                1. Else if _lastChunkHandling_ is *"loose"*, then
+                  1. If _chunkLength_ is 1, then
+                    1. Let _error_ be a new *SyntaxError* exception.
+                    1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+                  1. Set _bytes_ to the list-concatenation of _bytes_ and ! DecodeBase64Chunk(_chunk_, *false*).
+                1. Else,
+                  1. Assert: _lastChunkHandling_ is *"strict"*.
+                  1. Let _error_ be a new *SyntaxError* exception.
+                  1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+              1. Return the Record { [[Read]]: _length_, [[Bytes]]: _bytes_, [[Error]]: ~none~ }.
+            1. Let _char_ be the substring of _string_ from _index_ to _index_ + 1.
+            1. Set _index_ to _index_ + 1.
+            1. If _char_ is *"="*, then
+              1. If _chunkLength_ < 2, then
+                1. Let _error_ be a new *SyntaxError* exception.
+                1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+              1. Set _index_ to SkipAsciiWhitespace(_string_, _index_).
+              1. If _chunkLength_ = 2, then
+                1. If _index_ = _length_, then
+                  1. If _lastChunkHandling_ is *"stop-before-partial"*, then
+                    1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: ~none~ }.
+                  1. Let _error_ be a new *SyntaxError* exception.
+                  1. Return the Record { [[Read]]: _read_, [[Bytes]]: _bytes_, [[Error]]: _error_ }.
+                1. Set _char_ to the substring of _string_ from _index_ to _index_ + 1.
+                1. If _char_ is *"="*, then
+                  1. Set _index_ to SkipAsciiWhitespace(_string_, _index_ + 1).
+              1. If _index_ < _length_, then
```

Comment:
```suggestion
              1. If _index_ &lt; _length_, then
```

There are 5 occurrences of this that should be fixed.